docs/protocol.md

```diff
@@ -337,7 +337,7 @@ Two forms of string can be passed in the URI:
 
    The latest DID Document will be returned if found.
 
-1. DID with `initial-values` DID parameter: `did:sidetree:<unique-portion>;initial-values=<encoded-original-did-document>`
+2. DID with `initial-values` DID parameter: `did:sidetree:<unique-portion>;initial-values=<encoded-original-did-document>`
```

Comment:
Consider reverting this change, since MD will update the enumeration itself.

```suggestion
1. DID with `initial-values` DID parameter: `did:sidetree:<unique-portion>;initial-values=<encoded-original-did-document>`
```